src/components/Table/TableHead/TableHeader.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+import {
+  ArrowUp20 as Arrow,
+  ArrowsVertical20 as Arrows,
+} from '@carbon/icons-react';
+
+import { sortStates } from './state/sorting';
```

Comment:
this file is going to have to be duplicated if we are going to be using this TableHeader copy. Final approver should make sure there is an issue open to remove these files on updating to the Carbon package version where these changes will be found

Reply:
Yes, we will create an issue open to remove or refactor the temporary changes for updated Carbon version.
Our objective for this draft PR is to get it reviewed at high level/major suggestions, not sure we can convert this PR as normal, I am going to create new PR once development is completed with all major suggestion included and smooth working of resizing column, currently we are able to resize but that is not smooth as expected.